internal/pkg/manifest/lb_web_svc.go

```diff
@@ -327,3 +329,7 @@ func (e *Alias) ToString() string {
 	}
 	return strings.Join(e.StringSlice, ",")
 }
+
+func (s *LoadBalancedWebService) IsServiceAvailableInRegion(region string) (bool, error) {
+	return regions.IsServiceAvailable(ecs.EndpointsID, region)
```

Comment:
Can we move the constants to our `internal/aws` pkgs for example:
```go
// in internal/pkg/aws/ecs
package ecs

const ServiceID = ecs.ServiceID
```

We want to do this because it makes it easier to upgrade a dependency (aws-sdk-go) from v1 to v2. This way we have a single location to update `internal/pkg/aws` instead of updating multiple pkgs in the codebase. Hope that makes sense!

Reply:
(also I believe it needs to be `.ServiceID` instead of `.EndpointsID`)

Reply:
I believe `EndpointsID` is used here because of this description in the [aws-sdk-go doc](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#Endpoint.ServiceID):
> Deprecated: Use client package's EndpointsID value instead of these ServiceIDs. These IDs are not maintained, and are out of date.

Reply:
I went ahead and tested it - 
`apprunner` package's `ServiceID` is `"AppRunner"`, and its `EndpointsID` is `apprunner`. The `ServiceID` when passed into ` endpoints.RegionsForService` doesn't work, while the latter does.

Reply:
Yes even I did that testing and ServiceID did not work where as EndpintsID worked, hence used EndpointsID finally.